dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4055,7 +4055,11 @@ def test_roundtrip_decimal_dtype(tmpdir):
         ddf1 = ddf1.astype({"col1": pd.ArrowDtype(pa.decimal128(5, 2))})
     else:
         assert ddf1["col1"].dtype == ddf2["col1"].dtype
-    assert_eq(ddf1, ddf2, check_divisions=False)
+
+    # seems to be a Pyarrow bug
```

Comment:
Is there an issue for the bug?

Reply:
I have to ask Brock about this, not sure what the status here is